app/components/ui-table/cell/events/view/tickets/attendees/cell-action.js

```diff
@@ -1,5 +1,54 @@
 import classic from 'ember-classic-decorator';
 import Component from '@ember/component';
-
+import moment from 'moment';
 @classic
-export default class CellAction extends Component {}
+export default class CellAction extends Component {
+    currentlyCheckedin = [];
+    eventdates = [];
+    get allDates() {
+      const dates = [];
+      const date1 = moment(this.extraRecords.event.get('endsAtDate'));
+      const date2 = moment(this.extraRecords.event.get('startsAtDate'));
+      const diff = date1.diff(date2, 'days');
+      for (let i = 0; i < diff; i++) {
+        dates.push((moment(this.extraRecords.event.get('startsAtDate'), 'MM-DD-YYYY').add(i, 'days').format('MM-DD-YYYY')));
+        this.eventdates.push(moment(this.extraRecords.event.get('startsAtDate'), 'MM-DD-YYYY').add(i, 'days').format('MM-DD-YYYY'));
+      }
+      return dates;
+    }
+
+    get checkinTimes() {
+      if (this.extraRecords.checkinTimes) {
+        const x = this.extraRecords.checkinTimes?.split(',');
+        const checkinDates = [];
+        for (const i of x) {
+          checkinDates.push(moment(i).format('MM-DD-YYYY'));
+        }
+        return checkinDates;
+      }
+      return [];
+    }
+
+    get checkoutTimes() {
```

Comment:
@maze-runnar  shouldn't these all be computed properties?

Reply:
No